IdentityCore/src/broker_operation/response/MSIDBrokerOperationAccountResponse.h

```diff
@@ -0,0 +1,21 @@
+//
+//  MSIDBrokerOperationAccountResponse.h
+//  IdentityCore
+//
+//  Created by JZ on 10/1/19.
+//  Copyright © 2019 Microsoft. All rights reserved.
+//
+
+#import "MSIDBrokerOperationResponse.h"
+
+@class MSIDAccount;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSIDBrokerOperationAccountResponse : MSIDBrokerOperationResponse
```

Comment:
nit: should we call it "getAccountsResponse" to follow the naming of the request?

Reply:
:+1: changed :)